source/expectations/ut_expectation.tpb

```diff
@@ -679,5 +684,25 @@ create or replace type body ut_expectation as
     self.not_to(  ut_be_less_than (a_expected) );
   end;
 
+  member procedure to_include(self in ut_expectation, a_expected sys_refcursor) is
+  begin
+    self.to_( ut_include(a_expected) );
+  end;
+  
+  member procedure to_contain(self in ut_expectation, a_expected sys_refcursor) is
+  begin
+    self.to_( ut_include(a_expected) );
+  end;
+  
+  member procedure not_to_include(self in ut_expectation, a_expected sys_refcursor) is
+  begin
+   self.not_to( ut_include(a_expected).negated );
+  end;
+  
+  member procedure not_to_contain(self in ut_expectation, a_expected sys_refcursor) is
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this weak REF CURSOR by one with a return type. ([plsql:WeakRefCursorCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3AWeakRefCursorCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=774&issues=AWbTzT-qO08i52vd_XcQ&open=AWbTzT-qO08i52vd_XcQ)

[ISSUE_KEY]: AWbTzT-qO08i52vd_XcQ